go.mod

```diff
@@ -64,3 +64,5 @@ require (
 	google.golang.org/protobuf v1.25.0
 	gotest.tools/v3 v3.0.3 // indirect
 )
+
+replace git.fd.io/govpp.git => /home/ubuntu/Go_workspace/src/git.fd.io/govpp
```

Comment:
You should not use a local replace here.. instead you could push the changes from open review into your fork of govpp repo on github and replace like:

```
replace git.fd.io/govpp.git => github.com/mhalaj1/govpp e04017a413a7d6121059778057b20b22ce3aef15
```